### ISSUE TITLE
Add remote reading to reading index files

### ISSUE THREAD

**Issue by cgrdn** [ArgoCanada/bgcArgoDMQC#31]
Index files are read using pandas, which already has the capacity to read gzipped csv files from the web simply using:

```python
import pandas as pd
argo_index = pd.read_csv('ftp://ftp.ifremer.fr/ifremer/argo/ar_index_global_meta.txt.gz', compression='gzip',header=8)
```

This of course depends on you being connected to the internet *while* you are using the module, so should not remove the local reading of the index, but make remote reading the default (as it will always be up-to-date), with some conditions on network connection.

**Comment by cgrdn** [comment]
Updated with a try/except statement to use the local index if there is no internet connection. Resolved in c3919da, closing issue.

**Comment by cgrdn** [comment]
Keeping the issue closed, but turns out this is quite slow. Added an optional argument `remote=False` by default but still allows the user to read the index remotely so its not a waste of code.
